src/utils/api/api.ts

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import { Assets } from '@cognite/sdk';
+import { VAsset, VApiAssetList } from 'utils/validators';
+import { extractValidStrings } from 'utils/utils';
+
+const ocrURL = 'https://opin-api.cognite.ai/do_ocr';
```

Comment:
Hm. I am unsure if we should rely on this non-public api-endpoint for doing OCR. Anything preventing us from doing the call to Google's Vision API directly? Exposing secret key?